examples/kubernetes/nginx-deployment-dynamic-volume.yaml

```diff
@@ -0,0 +1,118 @@
+# Nginx pod with dynamic storage creation using NexentaStor Block CSI driver
+#
+# $ kubectl apply -f examples/kubernetes/nginx-dynamic-volume.yaml
+#
+# --------------------------------------
+# NexentaStor CSI Driver - Storage Class
+# --------------------------------------
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: nexentastor-block-csi-driver-sc-nginx-dynamic
+provisioner: nexentastor-block-csi-driver.nexenta.com
+allowVolumeExpansion: true
+---
+
+# ------------------------------------------------
+# NexentaStor CSI Driver - Persistent Volume Claim
+# ------------------------------------------------
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nexentastor-block-csi-driver-pvc-nginx-dynamic
+spec:
+  storageClassName: nexentastor-block-csi-driver-sc-nginx-dynamic
+  accessModes:
```

Comment:
For access modes can we please put comments here for users to use only one property at a time just for info(either 'ReadWriteMany' or 'ReadWriteOnce').

Reply:
There is also a ReadOnlyMany option. I can add that comment, but those are general kubernetes options, not sure that we need to document all of them.

Reply:
ok, if it's a standard Kubernetes command we can skip it.